site/downloads.markdown

```diff
@@ -9,168 +9,92 @@ isDownloads: true
 This page describes the installation of the Haskell toolchain, which consists of the following tools:
 
 *   [GHC](https://www.haskell.org/ghc/): the Glasgow Haskell Compiler
-
 *   [cabal-install](https://cabal.readthedocs.io): the Cabal installation tool for managing Haskell software
-
 *   [Stack](https://docs.haskellstack.org): a cross-platform program for developing Haskell projects
-
 *   [haskell-language-server](https://github.com/haskell/haskell-language-server) (optional): A language server for developers to integrate with their editor/IDE
 
-## Installation instructions { #ghcup }
+## Recommended installation instructions
```

Comment:
I wonder if we might not want to add something like `(All Operating Systems)` at the end here so someone glancing at the document will start here. I guess perhaps the `recommended` covers it, but I'd like to draw eyes to this section rather than the distro packages, since I think on average people will have a better experience avoiding the distro packages.